components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -230,6 +230,10 @@ open class GleanInternalAPI internal constructor () {
         StringsStorageEngine.record(GleanInternalMetrics.deviceModel, Build.MODEL)
         StringsStorageEngine.record(GleanInternalMetrics.architecture, Build.SUPPORTED_ABIS[0])
 
+        if (configuration.channel != null) {
```

Comment:
Minor point: I think if you use the `?.let` syntax here, the `orEmpty()` won't be necessary (since Kotlin enforces it's non-null within the block).

Reply:
Great point. I think this also fixes the low test coverage :)